blocks/links-bar-block/package.json

```diff
@@ -29,8 +27,7 @@
   "peerDependencies": {
     "@arc-fusion/prop-types": "^0.1.5",
     "@wpmedia/engine-theme-sdk": "*",
```

Comment:
Do we need this peer dependency?
cc: @JackHowa

Reply:
This should be removed.